.github/scripts/cmd/cmd.py

```diff
@@ -0,0 +1,204 @@
+if args.command == 'bench':
+    tempdir = tempfile.TemporaryDirectory()
+    print(f'Created temp dir: {tempdir.name}')
+    runtime_pallets_map = {}
+    failed_benchmarks = {}
+    successful_benchmarks = {}
+
+    profile = "release"
+
+    print(f'Provided runtimes: {args.runtime}')
+    # convert to mapped dict
+    runtimesMatrix = list(filter(lambda x: x['name'] in args.runtime, runtimesMatrix))
+    runtimesMatrix = {x['name']: x for x in runtimesMatrix}
+    print(f'Filtered out runtimes: {runtimesMatrix}')
+
+    # loop over remaining runtimes to collect available pallets
+    for runtime in runtimesMatrix.values():
+        os.system(f"forklift cargo build -p {runtime['package']} --profile {profile} --features runtime-benchmarks")
+        print(f'-- listing pallets for benchmark for {runtime["name"]}')
+        wasm_file = f"target/{profile}/wbuild/{runtime['package']}/{runtime['package'].replace('-', '_')}.wasm"
+        output = os.popen(
+            f"frame-omni-bencher v1 benchmark pallet --no-csv-header --all --list --runtime={wasm_file}").read()
+        raw_pallets = output.split('\n')
+
+        all_pallets = set()
+        for pallet in raw_pallets:
+            if pallet:
+                all_pallets.add(pallet.split(',')[0].strip())
+
+        pallets = list(all_pallets)
+        print(f'Pallets in {runtime}: {pallets}')
+        runtime_pallets_map[runtime['name']] = pallets
+
+    # filter out only the specified pallets from collected runtimes/pallets
+    if args.pallet:
+        print(f'Pallet: {args.pallet}')
+        new_pallets_map = {}
+        # keep only specified pallets if they exist in the runtime
+        for runtime in runtime_pallets_map:
+            if set(args.pallet).issubset(set(runtime_pallets_map[runtime])):
+                new_pallets_map[runtime] = args.pallet
+
+        runtime_pallets_map = new_pallets_map
+
+    print(f'Filtered out runtimes & pallets: {runtime_pallets_map}')
+
+    if not runtime_pallets_map:
+        if args.pallet and not args.runtime:
+            print(f"No pallets [{args.pallet}] found in any runtime")
+        elif args.runtime and not args.pallet:
+            print(f"{args.runtime} runtime does not have any pallets")
+        elif args.runtime and args.pallet:
+            print(f"No pallets [{args.pallet}] found in {args.runtime}")
+        else:
+            print('No runtimes found')
+        sys.exit(0)
+
+    header_path = os.path.abspath('./.github/scripts/cmd/file_header.txt')
+
+    for runtime in runtime_pallets_map:
+        for pallet in runtime_pallets_map[runtime]:
+            config = runtimesMatrix[runtime]
+            print(f'-- config: {config}')
+            if runtime == 'dev':
+                # to support sub-modules (https://github.com/paritytech/command-bot/issues/275)
+                search_manifest_path = f"cargo metadata --format-version 1 --no-deps | jq -r '.packages[] | select(.name == \"{pallet.replace('_', '-')}\") | .manifest_path'"
```

Comment:
```suggestion
                search_manifest_path = f"cargo metadata --locked --format-version 1 --no-deps | jq -r '.packages[] | select(.name == \"{pallet.replace('_', '-')}\") | .manifest_path'"
```
Otherwise it can update dependencies.